apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -113,65 +114,61 @@ private void handleRequest(HttpServletRequest request) {
     // Record the request type.
     BfdMDC.put(BfdMDC.computeMDCKey(MDC_PREFIX, "request_type"), request.getClass().getName());
 
-    // Set the default Operation (will hopefully be customized further in specific handler methods).
-    CanonicalOperation operation = new CanonicalOperation(CanonicalOperation.Endpoint.OTHER);
+    // Record the basic request components.
+    CanonicalOperation operation =
+        new CanonicalOperation(CanonicalOperation.Endpoint.matchByHttpUri(request));
+    BfdMDC.put(
+        BfdMDC.computeMDCKey(MDC_PREFIX, REQUEST_PREFIX, "http_method"), request.getMethod());
+    String url = replaceAllMbis(request.getRequestURL().toString());
+    BfdMDC.put(BfdMDC.computeMDCKey(MDC_PREFIX, REQUEST_PREFIX, "url"), url);
 
-    if (request instanceof HttpServletRequest) {
```

Comment:
I removed this check because it serves no purpose - the argument is already an `HttpServletRequest` so this check serves no purpose. Must've been left over from a previous refactor or something.